server/battles/BattleResultProcessor.h

```diff
@@ -46,19 +47,21 @@ struct FinishingBattleHelper
 
 	inline bool isDraw() const {return winnerSide == BattleSide::NONE;}
 
-	const CGHeroInstance *winnerHero, *loserHero;
+	ObjectInstanceID winnerId = ObjectInstanceID::NONE, loserId = ObjectInstanceID::NONE;
```

Comment:
- 1 declaration per line
- identifiers have default constructor that would initialize them to NONE

```suggestion
	ObjectInstanceID winnerId;
	ObjectInstanceID loserId;
```